c_src/eleveldb.cc

```diff
@@ -860,10 +861,11 @@ async_iterator_move(
         // using compare_and_swap has a hardware locking "set only if still in same state as before"
         //  (this is an absolute must since worker thread could change to false if
         //   hits end of key space and its execution overlaps this block's execution)
+	int cas_temp((eleveldb::MoveTask::PREFETCH_STOP != action )  // needed for Solaris CAS
```

Comment:
Please fix the indentation on these lines. The inserted lines (864, 865, and 868) use tab characters, but the surrounding code uses spaces.